csharp.test/TestParquetFileReader.cs

```diff
@@ -55,7 +55,7 @@ public static void TestFileHandleHasBeenReleased()
                 }
             });
 
-            StringAssert.StartsWith("Unable to cast object of type", exception?.Message);
+            StringAssert.StartsWith("Tried to get a LogicalColumnReader", exception?.Message);
```

Comment:
nit: NUnit has also fluent assertion syntax, I find it much nicer, e.g.:
```
Assert.That(exception?.Message, Does.StartWith("Tried to get a LogicalColumnReader"));
```
On the other hand, consistency is also important so it is probably good as it is now.

Reply:
Good point. Most new tests are being written with the fluent assertion syntax so consistency is already gone, so I've taken this opportunity to update all the uses of `StringAssert`. I've also added the column name to the message too.